test/expected/function_calls.out

```diff
@@ -0,0 +1,1925 @@
+                         "name": "uuidIdentity",                       +
+                         "type": {                                     +
+                             "kind": "SCALAR"                          +
+                         },                                            +
+                         "description": null                           +
+                     }                                                 +
+                 ]                                                     +
+             }                                                         +
+         }                                                             +
+     }                                                                 +
+ }
+(1 row)
+
+    rollback to savepoint a;
+    create table account(
+        id serial primary key,
+        email varchar(255) not null
+    );
+    create function returns_account()
+        returns account language sql stable
+    as $$ select id, email from account; $$;
+    insert into account(email)
+    values
+        ('aardvark@x.com'),
+        ('bat@x.com'),
+        ('cat@x.com');
+    comment on table account is e'@graphql({"totalCount": {"enabled": true}})';
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            returnsAccount {
+                id
+                email
+                nodeId
+                __typename
+            }
+        }
+    $$));
+                       jsonb_pretty                        
+-----------------------------------------------------------
+ {                                                        +
+     "data": {                                            +
+         "returnsAccount": {                              +
+             "id": 1,                                     +
+             "email": "aardvark@x.com",                   +
+             "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDFd",+
+             "__typename": "Account"                      +
+         }                                                +
+     }                                                    +
+ }
+(1 row)
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            returnsAccount {
+                email
+                nodeId
+            }
+        }
+    $$));
+                       jsonb_pretty                       
+----------------------------------------------------------
+ {                                                       +
+     "data": {                                           +
+         "returnsAccount": {                             +
+             "email": "aardvark@x.com",                  +
+             "nodeId": "WyJwdWJsaWMiLCAiYWNjb3VudCIsIDFd"+
+         }                                               +
+     }                                                   +
+ }
+(1 row)
+
+    select jsonb_pretty(graphql.resolve($$
+        query {
+            returnsAccount
```

Comment:
Right, this query should have been rejected. Not fixing it in this PR because of the following reaons:

1. After a bit of testing, I found that this is a general problem not specific to this PR, so I have created [an issue to track this bug](https://github.com/supabase/pg_graphql/issues/412). 
2. It can be worked around easily by not issuing the invalid query.

I've removed the invalid query from the tests.